exporters/openvpn_exporter.go

```diff
@@ -338,26 +488,63 @@ func (e *OpenVPNExporter) collectStatusFromFile(statusPath string, ch chan<- pro
 	return e.collectStatusFromReader(statusPath, conn, ch)
 }
 
+func (e *OpenVPNExporter) collectStatusFromApi(statusPath string, ch chan<- prometheus.Metric) error {
+	cmd := exec.Command("/usr/local/openvpn_as/scripts/sacli", "VPNStatus")
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+	if err != nil {
```

Comment:
I suppose this causes whole process die when `./sacli VPNStatus` returns error. It smells it's not correct.